[Jenkins Keycloak CI] - Cookies Tests - KcOidcBrokerLdapTest

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 36.188 s <<< FAILURE! - in org.keycloak.testsuite.broker.KcOidcBrokerLdapTest
[ERROR] org.keycloak.testsuite.broker.KcOidcBrokerLdapTest.testUpdateProfileOnFirstLogin  Time elapsed: 12.616 s  <<< FAILURE!
java.lang.AssertionError: Expected LoginPage but was Kerberos Unsupported (https://auth-server-127-0-0-1.nip.io:8543/auth/realms/consumer/protocol/openid-connect/auth?response_type=code&client_id=broker-app&redirect_uri=https%3A%2F%2Fauth-server-127-0-0-1.nip.io%3A8543%2Fauth%2Frealms%2Fmaster%2Fapp%2Fauth&state=ca5b65e2-816f-4162-9c2f-93ad72814682&scope=openid)
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.assertTrue(Assert.java:42)
        at org.keycloak.testsuite.pages.LoginPage.assertCurrent(LoginPage.java:215)
        at org.keycloak.testsuite.pages.LoginPage.open(LoginPage.java:259)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:569)
        at org.jboss.arquillian.graphene.proxy.GrapheneProxyHandler.invokeReal(GrapheneProxyHandler.java:143)
        at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$1.invoke(GrapheneContextualHandler.java:180)
        at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$2.call(GrapheneContextualHandler.java:252)
        at org.jboss.arquillian.graphene.context.BrowserActions.performAction(BrowserActions.java:62)
        at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.invoke(GrapheneContextualHandler.java:248)
        at org.jboss.arquillian.graphene.proxy.GrapheneProxyHandler.intercept(GrapheneProxyHandler.java:106)
        at org.jboss.arquillian.graphene.proxy.GrapheneProxyHandler.intercept(GrapheneProxyHandler.java:81)
        at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.intercept(GrapheneContextualHandler.java:92)
        at org.keycloak.testsuite.pages.LoginPage_ByGraphene.open(Unknown Source)
        at org.keycloak.testsuite.broker.KcOidcBrokerLdapTest.testUpdateProfileOnFirstLogin(KcOidcBrokerLdapTest.java:72)
```

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

`mvn -f testsuite/integration-arquillian/tests/base/pom.xml clean install -Pauth-server-quarkus -Pfirefox-strict-cookies -Dauth.server.host=auth-server-127-0-0-1.nip.io -Dauth.server.host2=127.0.0.1.sslip.io -Dtest=KcOidcBrokerLdapTest#testUpdateProfileOnFirstLogin`

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcOidcBrokerLdapTest.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcOidcBrokerLdapTest.java
@@ -26,24 +26,16 @@ import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Test;
 import org.keycloak.common.util.MultivaluedHashMap;
-import org.keycloak.federation.kerberos.CommonKerberosConfig;
 import org.keycloak.models.LDAPConstants;
 import org.keycloak.models.utils.ModelToRepresentation;
 import org.keycloak.representations.idm.ComponentRepresentation;
 import org.keycloak.storage.UserStorageProvider.EditMode;
 import org.keycloak.storage.UserStorageProviderModel;
 import org.keycloak.storage.ldap.LDAPStorageProviderFactory;
-import org.keycloak.storage.ldap.kerberos.LDAPProviderKerberosConfig;
-import org.keycloak.testsuite.KerberosEmbeddedServer;
 import org.keycloak.testsuite.admin.ApiUtil;
-import org.keycloak.testsuite.federation.kerberos.KeycloakSPNegoSchemeFactory;
-import org.keycloak.testsuite.util.KerberosRule;
+import org.keycloak.testsuite.util.LDAPRule;
 
 public final class KcOidcBrokerLdapTest extends AbstractInitializedBaseBrokerTest {
-
-    private static final String PROVIDER_CONFIG_LOCATION = "classpath:kerberos/kerberos-ldap-connection.properties";
-
-    private KeycloakSPNegoSchemeFactory spnegoSchemeFactory;
 
     @Override
     protected BrokerConfiguration getBrokerConfiguration() {
@@ -51,13 +43,10 @@ public final class KcOidcBrokerLdapTest extends AbstractInitializedBaseBrokerTes
     }
 
     @ClassRule
-    public static KerberosRule kerberosRule = new KerberosRule(PROVIDER_CONFIG_LOCATION, KerberosEmbeddedServer.DEFAULT_KERBEROS_REALM);
+    public static LDAPRule ldapRule = new LDAPRule();
 
     @Before
     public void onBefore() {
-        getKerberosRule().setKrb5ConfPath(testingClient.testing());
-        spnegoSchemeFactory = new KeycloakSPNegoSchemeFactory(getKerberosConfig());
-        oauth.clientId("kerberos-app");
         ComponentRepresentation rep = getUserStorageConfiguration();
         Response resp = adminClient.realm(bc.consumerRealmName()).components().add(rep);
         getCleanup().addComponentId(ApiUtil.getCreatedId(resp));
@@ -76,11 +65,11 @@ public final class KcOidcBrokerLdapTest extends AbstractInitializedBaseBrokerTes
     }
 
     private ComponentRepresentation getUserStorageConfiguration(String providerName, String providerId) {
-        Map<String,String> kerberosConfig = getKerberosRule().getConfig();
-        kerberosConfig.put(LDAPConstants.SYNC_REGISTRATIONS, "false");
-        kerberosConfig.put(LDAPConstants.EDIT_MODE, EditMode.UNSYNCED.name());
-        kerberosConfig.put(UserStorageProviderModel.IMPORT_ENABLED, "true");
-        MultivaluedHashMap<String, String> config = toComponentConfig(kerberosConfig);
+        Map<String,String> ldapConfig = ldapRule.getConfig();
+        ldapConfig.put(LDAPConstants.SYNC_REGISTRATIONS, "false");
+        ldapConfig.put(LDAPConstants.EDIT_MODE, EditMode.UNSYNCED.name());
+        ldapConfig.put(UserStorageProviderModel.IMPORT_ENABLED, "true");
+        MultivaluedHashMap<String, String> config = toComponentConfig(ldapConfig);
 
         UserStorageProviderModel model = new UserStorageProviderModel();
         model.setLastSync(0);
@@ -103,16 +92,7 @@ public final class KcOidcBrokerLdapTest extends AbstractInitializedBaseBrokerTes
         return config;
     }
 
-    private KerberosRule getKerberosRule() {
-        return kerberosRule;
-    }
-
-
-    private CommonKerberosConfig getKerberosConfig() {
-        return new LDAPProviderKerberosConfig(getUserStorageConfiguration());
-    }
-
     private ComponentRepresentation getUserStorageConfiguration() {
-        return getUserStorageConfiguration("kerberos-ldap", LDAPStorageProviderFactory.PROVIDER_NAME);
+        return getUserStorageConfiguration("ldap", LDAPStorageProviderFactory.PROVIDER_NAME);
     }
 }

